cgmanifest.json

```diff
@@ -6516,6 +6516,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "intel-pf-bb-config",
+          "version": "21.11.0",
```

Comment:
Looks like this version has an extra `.0` compared to the actual package version